templates/makefile-c++/README.md

```diff
@@ -0,0 +1,38 @@
+# C++ Makefile Project Template
+
+## About this Project
+This is a basic makefile template designed for C++. 
+
+### Supported Platforms 
+The contents of the makefile are generalized to support compilation to target either Windows or Linux out of the box. You may need to include platform specific headers as you go, which may restrict your project to a single platform. Take some time to consider what platforms you plan to target, or if cross-compatibility is required.
+
+__*Note: This project has been only tested for Windows using the Git Bash command line interface. Using PowerShell, cmd.exe, or other, non-unix shells may result in errors using make and g++.*__
+
+For C++ projects targeting embedded ARM systems, please use a platformIO template like the [Nucleo F446RE template](https://github.com/qfsae/zenith/tree/master/templates/nucleo-f446re).
+
+## How to use this Template 
+To use this template, simply copy the directory `templates/makefile-c++/` into the root directory of the repository. 
```

Comment:
Consider noting that the directory named should be changed to match the project being worked on?

Reply:
Also think that this directory can be abbreviated to `cpp`or something. I think `makefile-c++`is a little mutch

Reply:
Renaming to makefile-cpp - I can see a time when we have non-makefile oriented C++ templates, and it'll just get renamed once in use.